src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1452,95 +1452,9 @@ export default class CanvasController {
      * @private
      */
     _drawWindVane(cc) {
-        cc.save();
-        cc.font = 'bold 10px monoOne, monospace';
-        cc.translate(
-            CanvasStageModel.width,
-            CanvasStageModel.height
-        );
-
-        const airportModel = AirportController.airport_get();
-        const size = 80;
-        const size2 = size / 2;
-        const padding = 16;
-        const dot = 16;
-        let windspeed_line;
-        let highwind;
-
-        // Shift compass location
-        cc.translate(-size2 - padding, -size2 - padding);
-        cc.lineWidth = 4;
-
-        // Outer circle
-        cc.fillStyle = this.theme.WIND_VANE.OUTER_RING_FILL;
-        cc.beginPath();
-        cc.arc(0, 0, size2, 0, tau());
-        cc.fill();
-
-        // Inner circle
-        cc.lineWidth = 1;
-        cc.beginPath();
-        cc.arc(0, 0, dot / 2, 0, tau());
-        cc.strokeStyle = this.theme.WIND_VANE.INNER_RING_STROKE;
-        cc.stroke();
-
-        // Wind Value
-        cc.fillStyle = this.theme.WIND_VANE.WIND_SPEED_TEXT;
-        cc.textAlign = 'center';
-        cc.textBaseline = 'center';
-        cc.font = '9px monoOne, monospace';
-        cc.fillText(airportModel.wind.speed, 0, 3.8);
-        cc.font = 'bold 10px monoOne, monospace';
-
-        // Wind line
-        if (airportModel.wind.speed > 8) {
-            windspeed_line = airportModel.wind.speed / 2;
-            highwind = true;
-        } else {
-            windspeed_line = airportModel.wind.speed;
-            highwind = false;
-        }
-
-        cc.save();
-        cc.translate(
-            -dot / 2 * sin(airportModel.wind.angle),
-            dot / 2 * cos(airportModel.wind.angle)
-        );
+        // left over from old `._drawWindVane()` method. removing this, or moving it to `._drawAirspaceAndRangeRings`
+        // or `._drawRunwayLabels()` causes a white circle to be drawn with the airport center at its origin
         cc.beginPath();
```

Comment:
why this still needs to be here is beyond me.  I can't for the life of me figure out why this still needs to be here.  Moving this to either method called before or after this results in the same white circle.